app/models/challenge.rb

```diff
@@ -42,6 +42,23 @@ def featured_contributions
     end
   end
 
+  def latest_contributions
+    ordering = "published_at DESC"
+
+    case active_stage
+    when 'experience'
+      experience_stage.experiences.published.order_by(ordering).first(2)
```

Comment:
Got it - thanks. That's the only place I was using `unscope()` -- it was for the re-ordering that we built back in the Spring. I edited `sortable.rb` to `reorder` instead of `unscope`.